docs/2.deploy/20.providers/gitlab-pages.md

```diff
@@ -0,0 +1,43 @@
+# GitLab Pages
+
+> Deploy Nitro apps to GitLab Pages.
+
+**Preset:** `gitlab_pages`
+
+:read-more{title="GitLab Pages" to="https://pages.github.com/"}
+
+## Setup
+
+Follow the steps to [create a GitLab Pages site](https://docs.gitlab.com/ee/user/project/pages/#getting-started).
+
+## Deployment
+
+1. Here is an example GitLab Pages workflow to deploy your site to GitLab Pages:
+
+```yaml [.gitlab-ci.yml]
+# The Docker image that will be used to build your app
+image: node:lts
+# Functions that should be executed before the build script is run
+before_script:
+  - npm install
```

Comment:
This way we can respect package-manager for free


```suggestion
  - npx nypm install
```

Reply:
In GitHub Pages we say '# Pick your own package manager and build script', and also in a variety of other providers. Should I update them as well?

https://github.com/unjs/nitro/blob/97e2a316c114b6196323354d73bf0b5f34c207f3/docs/2.deploy/20.providers/github-pages.md#L37